src/App.tsx

```diff
@@ -1,167 +1,62 @@
-                  </td>
-                </tr>
-
-                <tr data-cy="person" className="has-background-warning">
-                  <td>
-                    <a href="#/people/jan-frans-van-brussel-1761">
-                      Jan Frans van Brussel
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+              Home
+            </NavLink>
+
+            <NavLink
+              className={({ isActive }) => {
+                return cn('navbar-item', {
+                  'has-background-grey-lighter': isActive,
+                });
+              }}
+              to="/people"
+            >
+              People
+            </NavLink>
           </div>
         </div>
-      </div>
-    </main>
-  </div>
-);
+      </nav>
+
+      <main className="section">
+        <div className="container">
+          <Routes>
+            <Route path="/">
+              <Route index element={<h1 className="title">Home Page</h1>} />
+              <Route path="/home" element={<Navigate to="/" replace />} />
+              <Route path="people" element={<PeopleList />}>
+                <Route index />
+                <Route path=":personSlug?" />
+              </Route>
+            </Route>
```

Comment:
The nested `Route` structure under the path `/` is not necessary for the `HomePage`. You can directly define the `Route` for the `HomePage` without nesting it under another `Route`. This might simplify the routing logic.